dev-docs/modules/fledgeForGpt.md

```diff
@@ -2,172 +2,63 @@
 
-This module allows Prebid.js to support FLEDGE by integrating it with GPT's [experimental FLEDGE
-support](https://github.com/google/ads-privacy/tree/master/proposals/fledge-multiple-seller-testing).
+This module allows Prebid.js to support PAAPI (formerly FLEDGE) by integrating with GPT's support for [component auctions](https://developers.google.com/publisher-tag/reference#googletag.config.componentauctionconfig).
 
-To learn more about FLEDGE in general, go [here](https://github.com/WICG/turtledove/blob/main/FLEDGE.md).
-
-This document covers the steps necessary for publishers to enable FLEDGE on their inventory. It also describes
-the changes Bid Adapters need to implement in order to support FLEDGE.
+To learn more about PAAPI in general, go [here](https://github.com/WICG/turtledove/blob/main/FLEDGE.md).
 
 ## Publisher Integration
 
-Publishers wishing to enable FLEDGE support must do two things. First, they must compile Prebid.js with support for this module.
-This is accomplished by adding the `fledgeForGpt` module to the list of modules they are already using:
-
-```bash
-gulp build --modules=fledgeForGpt,...
-```
-
-Second, they must enable FLEDGE in their Prebid.js configuration.
-This is done through module level configuration, but to provide a high degree of flexiblity for testing, FLEDGE settings also exist at the bidder level and slot level.
-
-### Module Configuration
-
-This module exposes the following settings:
-
-{: .table .table-bordered .table-striped }
-|Name |Type |Description |Notes |
-| ------------ | ------------ | ------------ |------------ |
-|enabled | Boolean |Enable/disable the module |Defaults to `false` |
-|bidders | Array[String] |Optional list of bidders |Defaults to all bidders |
-|defaultForSlots | Number |Default value for `imp.ext.ae` in requests for specified bidders |Should be 1 |
-
-As noted above, FLEDGE support is disabled by default. To enable it, set the `enabled` value to `true` for this module and configure `defaultForSlots` to be `1` (meaning _Client-side auction_).
-using the `setConfig` method of Prebid.js. Optionally, a list of
-bidders to apply these settings to may be provided:
-
-```js
-pbjs.que.push(function() {
-  pbjs.setConfig({
-    fledgeForGpt: {
-      enabled: true,
-      bidders: ['openx', 'rtbhouse'],
-      defaultForSlots: 1
-    }
-  });
-});
-```
-
-### Bidder Configuration
-
-This module adds the following setting for bidders:
+To use PAAPI with GPT:
 
-{: .table .table-bordered .table-striped }
-|Name |Type |Description |Notes |
-| ------------ | ------------ | ------------ |------------ |
-| fledgeEnabled | Boolean | Enable/disable a bidder to participate in FLEDGE | Defaults to `false` |
-|defaultForSlots | Number |Default value for `imp.ext.ae` in requests for specified bidders |Should be 1|
+- include this module with your Prebid.js bundle; this also automatically includes the [PAAPI module](/dev-docs/modules/paapi.html)
```

Comment:
I find the the two similarly named modules confusing. Perhaps it is clear to folks that swim with modules regularly, but is there an explanation or diagram?

Reply:
I missed that flipping between source and rich diff.